packages/next/package.json

```diff
@@ -192,6 +192,7 @@
     "@types/ws": "8.2.0",
     "@vercel/ncc": "0.34.0",
     "@vercel/nft": "0.22.6",
+    "@vercel/turbopack-ecmascript-runtime": "https://gitpkg-fork.vercel.sh/vercel/turbo/crates/turbopack-ecmascript-runtime/js?e3913eef6",
```

Comment:
TODO: This needs to be updated once https://github.com/vercel/turbo/pull/5814 is released.